sim_data/plotter.py

```diff
@@ -14,7 +14,7 @@
 vy = []
 vz = []
 
-ax = []
+aX = []
```

Comment:
Why is this formatted differently?

Reply:
Also I'm now mostly used to reviews on bitbucket, not github, so I'm experimenting by making one a simple comment and this one is a review I think?

Reply:
Hehe the plotting stuff in python is a big mess right now. I'm not putting in any effort to make it clean or coherent.

But essentially the array `ax` was getting overwritten with `ax = fig.add_subplot()` plots further down. So I just did this to get a quick fix.